web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -1134,3 +1134,29 @@ function joinup_core_post_update_delete_orphaned_solutions() {
     $solution->delete();
   }
 }
+
+/**
+ * Implements hook_post_update_NAME().
```

Comment:
On (post)updates we write here a description of the operation we're performing. The first word should be an imperative verb (no "s" at the end of the word). See https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#updateN